go/consensus/tendermint/tendermint.go

```diff
@@ -257,6 +263,9 @@ func (t *tendermintService) Start() error {
 		}
 		go t.syncWorker()
 		go t.worker()
+		if viper.GetString(cmmetrics.CfgMetricsAddr) != "" {
```

Comment:
Make this based on the configured metrics mode instead and enable it if the mode is not none.